src/DynamoCore/DynamoCore.csproj

```diff
@@ -82,8 +82,8 @@ limitations under the License.
       <HintPath>..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="ProtoGeometry, Version=2.12.0.5475, Culture=neutral, processorArchitecture=AMD64">
-      <HintPath>..\packages\DynamoVisualProgramming.LibG_227_0_0.2.12.0.5475\lib\net48\ProtoGeometry.dll</HintPath>
+    <Reference Include="ProtoGeometry, Version=2.12.0.5535, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\packages\DynamoVisualProgramming.LibG_226_0_0.2.12.0.5535\lib\net48\ProtoGeometry.dll</HintPath>
```

Comment:
This should reference 227 nugets, not 226. It may not make a difference but we ensure that we are consistent with all projects referencing protogeometry from 227.